keanu-project/src/main/java/io/improbable/keanu/tensor/intgr/IntegerTensor.java

```diff
@@ -161,4 +161,16 @@ static IntegerTensor concat(int dimension, IntegerTensor... toConcat) {
 
     BooleanTensor greaterThanOrEqual(int value);
 
+    IntegerTensor min(IntegerTensor min);
+
+    IntegerTensor max(IntegerTensor max);
```

Comment:
don't you think it would be more natural for these to be static prefix operators (`min(a, b)`) rather than non-static infix operators (`a.min(b)`)?

Reply:
Yeah - updated these. Same for doubles.